java.source.base/test/unit/src/org/netbeans/api/java/source/SourceUtilsTestUtil.java

```diff
@@ -280,9 +281,13 @@ public TestProxyClassPathProvider(FileObject sourceRoot, FileObject buildRoot, F
         
         public ClassPath findClassPath(FileObject file, String type) {
             try {
-            if (ClassPath.BOOT == type || JavaClassPathConstants.MODULE_BOOT_PATH.equals(type)) {
-                return ClassPathSupport.createClassPath(getBootClassPath().toArray(new URL[0]));
-            }
+                if (ClassPath.BOOT == type) {
```

Comment:
The change was made to run TestCase in JDK10, otherwise getting error can't access package java.lang

Reply:
This looks good I think.